keras_cv/bounding_box/__init__.py

```diff
@@ -12,14 +12,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from keras_cv.bounding_box.convert_to_corners import convert_to_corners
+from keras_cv.bounding_box.converters import transform_format
 from keras_cv.bounding_box.pad_batch_to_shape import pad_batch_to_shape
 
-# These are the indexes used in Tensors to represent each corresponding side.
-LEFT, TOP, RIGHT, BOTTOM = 0, 1, 2, 3
+# per format axis selector constants
 
-# Regardless of format these constants are consistent.
-# Class is held in the 5th index
-CLASS = 4
-# Confidence exists only on y_pred, and is in the 6th index.
-CONFIDENCE = 5
+
+class XYXY:
+    LEFT = 0
+    TOP = 1
+    RIGHT = 2
+    BOTTOM = 3
+    CLASS = 4
+    CONFIDENCE = 5
+
```

Comment:
extended_corners format, 
one of the KPI needed it guess.

Reply:
This can be done in a follow up 👍

Reply:
yup, make sense, can be done in next pr :)